src/swap/components/SwapProvider.test.tsx

```diff
@@ -245,6 +251,22 @@ describe('SwapProvider', () => {
     expect(result.current.error).toBeUndefined();
   });
 
+  it('should reset inputs when setLifeCycleStatus is called with success', async () => {
+    const { result } = renderHook(() => useSwapContext(), { wrapper });
+    await act(async () => {
+      result.current.setLifeCycleStatus({
+        statusName: 'success',
+        statusData: { transactionReceipt: '0x123' },
+      });
+    });
+
```

Comment:
In the future let's avoid spaces in tests, I noticed that when reading them in a very long file, those spaces really make hard to parse many tests all at once.

Let's keep a space only between `it` but not inside an `it`.

Reply:
removed - thanks for catching this